Tasks/Maven/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,7 +6,7 @@
   "loc.group.displayName.junitTestResults": "JUnit Test Results",
   "loc.group.displayName.codeCoverage": "Code Coverage",
   "loc.group.displayName.advanced": "Advanced",
-  "loc.group.displayName.SQAnalysis": "SonarQube Analysis",
```

Comment:
The SonarQube UI options will be grouped with the new code analysis tools as part of a new "Code Analysis" UI section, as per the UI mockup.